input/data/2008/2008ApJ...679.1427A/tev-000014.yaml

```diff
@@ -0,0 +1,35 @@
+# Source ID from TeGeV-cat: http://www.asdc.asi.it/tgevcat/)
+# LS I +61 303
+# VER J0240+612
+source_id: 14
+# Reference ID from ADS
```

Comment:
Remove comment "Reference ID from ADS". We have this almost everywhere, no need to comment.